### PR TITLE
July kb articles

### PR DESCRIPTION
**Pull request by filipKovachev**
Removed the old KBs because they don't work properly. I am adding three new ones instead, runnable code for each can be found here:

Editor KB: https://stackblitz.com/edit/bfhxmj-zp443k?file=src%2Fmain.vue
DropDownList KB: https://stackblitz.com/edit/yqcm8f-vkpovr?file=src%2FValueRender.vue
ExcelExport KB: https://stackblitz.com/edit/ivqva1-8by7rc?file=src%2Fmain.vue

part of https://github.com/telerik/kendo-react-private/issues/4409

### CODE REVIEW

**Review by zdravkov — APPROVED** [review]

File: knowledge-base/upload-custom-validation-error.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Kendo UI for Vue Native Upload</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+This Knowledge base(KB) article demonstrates how we can add a custom validation error for the Upload component.
```
Comment (vveesseelliinnaa) [inline knowledge-base/upload-custom-validation-error.md:30]: Should we a sentence explaining when the added custom error will be shown in the demo?

File: knowledge-base/examples/chart-tooltip-series-item/main.vue
Comment (vveesseelliinnaa) [inline knowledge-base/examples/chart-tooltip-series-item/main.vue:1]: This sample does not work on Stackblitz for some reason (it is OK in CodeSandBox)

File: knowledge-base/examples/chart-tooltip-series-item/main.vue [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            aggregate: 'sum',
+            text: 'Cost: Actual. Total',
+          },
+        ],
+        type: 'bar',
+        datasetProcessed: [
+          {
+            Id: 31289,
+            ApprovalStatus: {
+              Name: 'Approved by a very log whatnot label here sooo long',
```
Comment (vveesseelliinnaa) [inline knowledge-base/examples/chart-tooltip-series-item/main.vue:108]: Make sure that the long message does not contain so many typos. E.g., you can change it to 
"Approved by a very long, loooong and even longer label shown inside a tooltip"

File: knowledge-base/chart-tooltip-series-item.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	    	<td>Product</td>
+	    	<td>Progress® Kendo UI for Vue Native</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to add a tooltip to the Chart series items?
```
Comment (vveesseelliinnaa) [inline knowledge-base/chart-tooltip-series-item.md:30]: It was not clear to me that the tooltip will be shown only when I hover over the labels in the y axis. Can you update the description to state that, so the clients will not expect them over the actual series items of the chart.

**Review by vveesseelliinnaa — CHANGES_REQUESTED** [review]

**Review by vveesseelliinnaa — APPROVED** [review]

**Pull request merged**